### ISSUE TITLE
Alerts not going away once acknowledged

### ISSUE THREAD

**Issue by ospfbgp** [librenms/librenms#4337]
I have been working on understanding alerts and ran into some hurdles when acknowledged via the gui.  Alerts are working as expected I get the email and the acknowledgement email.  Once I acknowledge the alert from the gui and it gets muted but it is never removed from the Notifications page.  I have to disable the alert for the alert to delete from the Notifications page.  Any help would be greatly appreciated.

Here is what I know....
I have checked the timezone on my ubuntu server and the php.ini files.  All are set to America/New_York.  I also checked mysql timezone.

root@librenms:/opt/librenms# mysql -V
mysql  Ver 15.1 Distrib 10.0.25-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
root@librenms:/opt/librenms# mysql
Welcome to the MariaDB monitor.  Commands end with ; or \g.
Your MariaDB connection id is 146
Server version: 10.0.25-MariaDB-0ubuntu0.16.04.1 Ubuntu 16.04

Copyright (c) 2000, 2016, Oracle, MariaDB Corporation Ab and others.

Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.

MariaDB [(none)]> select @@system_time_zone;
+--------------------+
| @@system_time_zone |
+--------------------+
| EDT                |
+--------------------+
1 row in set (0.00 sec)

Here is an alert that is stuck and will not go away from notifications or the alerts table.  

MariaDB [librenms]> select * from alerts;
+----+-----------+---------+-------+---------+------+---------------------+
| id | device_id | rule_id | state | alerted | open | timestamp           |
+----+-----------+---------+-------+---------+------+---------------------+
|  2 |         2 |      21 |     2 |       1 |    0 | 2016-09-04 09:23:02 |
+----+-----------+---------+-------+---------+------+---------------------+
1 row in set (0.00 sec)

<img width="1026" alt="events_alert_and_acknowledged" src="https://cloud.githubusercontent.com/assets/21048481/18231273/79b55214-7283-11e6-9d82-62abce61cadd.png">
<img width="1336" alt="notification_acknowledged" src="https://cloud.githubusercontent.com/assets/21048481/18231275/79d9b104-7283-11e6-81e1-9369dcb6b4ea.png">
<img width="610" alt="syslog_alert" src="https://cloud.githubusercontent.com/assets/21048481/18231274/79d2ddca-7283-11e6-865e-b0bf372bfc4d.png">

**Comment by ospfbgp** [comment]
After some testing I would click acknowledge with the alert disabled and notice alerts.open would change to 1.  This does not seem to happen when alert rule is enabled.  What would cause it not to change from alerts.open to change from 0 to 1 from the notifications gui and acknowledge is clicked? It appears once it is alerts.open have a value = 1 that when alerts.php is run it will purge it from the table.

MariaDB [librenms]> select * from alerts;
+----+-----------+---------+-------+---------+------+---------------------+
| id | device_id | rule_id | state | alerted | open | timestamp           |
+----+-----------+---------+-------+---------+------+---------------------+
|  2 |         2 |      21 |     2 |       1 |    1 | 2016-09-04 10:08:19 |
+----+-----------+---------+-------+---------+------+---------------------+
1 row in set (0.00 sec)

**Comment by laf** [comment]
Acknowledged alerts don't disappear from the alerts list - that's by design.

**Comment by ospfbgp** [comment]
I guess I need to create an active notifications page. 

Daniel Cox
danielcoxman@gmail.com
859-552-8091

**Comment by ospfbgp** [comment]
So what does it take for the alert rule to not be paused. It seems my alert rule stays paused even after I acknowledge.  I need to continue sending alerts after I acknowledge. 

Can you elaborate on how this should operate?  Just want to understand.

Thank you for the help. 

Daniel Cox
danielcoxman@gmail.com
859-552-8091

**Comment by laf** [comment]
The rule being paused is / shouldn't be anything to do with acknowledged alerts. The pause symbol means the alert is not currently active / running / live.

**Comment by ospfbgp** [comment]
laf,
I want to say I have found the problem  after a little troubleshooting I crafted my alert rule wrong.  You put me in the write direction by giving me information on how this works.  So my alert rule was always hitting because I did not put the % in front of macro.  I was looking for syslog.msg for the last 5 minutes that match the string "failed login".    Since I did not have the % my rule would hit and go to X put would never clear and go back to green/checked because I was trying to match forever and not 5 minutes.  I have fixed it.  It works as expected now.

thank you for the help.
I have correctly formatted rule now.

<img width="614" alt="screen shot 2016-09-04 at 9 58 35 pm" src="https://cloud.githubusercontent.com/assets/21048481/18235583/d4695b80-72ea-11e6-8746-ce897b340136.png">
